### PR TITLE
Add Google Analytics

### PR DESCRIPTION
**Pull request by johnnolan**
* Adds Google Analytics
* Adds cookie banner
* Adds cookie banner confirmations
* Adds cookie page

If you want to test you can follow the instructions here [https://github.com/ministryofjustice/opg-modernising-lpa-docs#setup](https://github.com/ministryofjustice/opg-modernising-lpa-docs#setup) to set it up locally and run

### CODE REVIEW

File: src/_includes/scripts/index.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+        toggleCookieBannerReject(false);
+      }
+      if (e.target.value === "hide") {
+        toggleCookieBannerConfirm(true);
+        toggleCookieBannerReject(true);
+      }
+    }
+
+    if (isAnalyticsCookieSet) {
+      if (e.target) {
```
Comment (mattmachell) [inline src/_includes/scripts/index.js:36]: What's the browser target? Might be able to simplify this with optional chaining.

Comment (johnnolan) [reply src/_includes/scripts/index.js:36]: Officially IE11 I believe but decided on making it all as simple as possible to increase browser coverage. Same reason I didn't go down a module route

File: src/_includes/scripts/index.js [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+    toggleCookieBannerConfirm(true);
+    toggleCookieBannerReject(true);
+    if (gaCookieValue == "true") {
+      setupGoogleAnalytics("G-70EPL8ZB7Y");
+    }
+  }
+
+  if (location.pathname === "/view-cookies/") {
+    if (gaCookieSettings === null || gaCookieValue == "false") {
+      document.querySelector(
```
Comment (mattmachell) [inline src/_includes/scripts/index.js:60]: Does it streamline this to put the selector in a string and run the querySelector outside the if.

Comment (johnnolan) [reply src/_includes/scripts/index.js:60]: Will look again tomorrow at this, think there is a better way but needing the true/false value of the radio is an issue.

**Comment by johnnolan** [comment]
@mattmachell tried to tidy it all up to, let me know your thoughts

**Review by mattmachell — APPROVED** [review]

**Pull request merged**